common/app/views/main.scala.html

```diff
@@ -1,10 +1,8 @@
 @(metaData: model.MetaData, projectName: Option[String] = None, curlPaths: Map[String, String] = Map())(head: Html)(body: Html)(implicit request: RequestHeader)
-
```

Comment:
Removed whitespace from main here as there were errors being thrown because there should be nothing above the ``<!DOCTYPE``